### PR TITLE
Debian packaging and deployment to GitHub releases

### LINKED ISSUE #29: Debian packaging
We have to look into [`cargo-deb`](https://github.com/mmstick/cargo-deb).

@syxolk Do you think dynamic linking to GNU `libc` is ok? Or should we compile with `musl`?

### PR DESCRIPTION
**Pull request by mp4096**
Closes #29

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,9 +6,24 @@ license = "MIT"
...
 readme = "README.md"
 keywords = ["latex"]
+repository = "https://github.com/mp4096/indentex"
 publish = false
 
 
+[package.metadata.deb]
+maintainer = "Mikhail Pak <mikhail.pak@tum.de>"
+copyright = "2016-2017, Mikhail Pak <mikhail.pak@tum.de>, Hans Kirchner <h.kirchner@tum.de>"
+license_file = ["LICENSE.md", "4"]
```
Comment (syxolk) [inline Cargo.toml]: What does the `"4"` mean?

Comment (syxolk) [reply Cargo.toml]: > The location of the license and the amount of lines to skip at the top

Nevermind...

Comment (mp4096) [reply Cargo.toml]: `0____0` I didn't read the docs and I was 100% sure that this is the numerical permission for the license file...

**Comment by mp4096** [comment]
Awww shit, I forgot to add a guard to use stable Rust only... and I think explicit target specification would be nice.

**Pull request merged**